service-manual/making-software/security-testing.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: detailed-guidance
+title: Security testing
+subtitle: What is security testing and how to use it
+category: making-software
+type: guide
+audience:
+  primary: developers, qa
+  secondary: web-ops, tech-archs
+status: draft
+phases:
+  - alpha
+  - beta
+  - live
+breadcrumbs:
+  -
+    title: Home
+    url: /service-manual
+  -
+  	title: Making software
```

Comment:
The build is failing due an indentation problem, need to remove two spaces from these two lines.